### ISSUE TITLE
Use workshop titles as alt texts for cover images on the Initiatives page when alt texts are not defined in Airtable

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/wecount.inclusivedesign.ca#436]
## Describe the bug

The lighthouse CI report reports the minscores of "accessibility" on the Initiatives page is lower than expected.

This is because some workshop images on the Initiatives page are missing alt texts.

Looking at the workshop data defined in the Airtable, some workshop images have default images uploaded but their alt texts are missing. The record creator was probably assuming default images are not workshop specific images so alt texts are not defined.

The solution is to improve the code to use the workshop title as the alt text for a workshop image when its alt text is missing from the Airtable.

## To Reproduce

1. Use Chrome to go to the Initiatives page such as https://dev--wecount.netlify.app/initiatives/;
2. Open dev tool -> lighthouse tab;
3. Click "generate report" and check report values
4. "accessibility" score is 98.

## Expected behavior

The "accessibility" score should be 100.
